database/Dockerfile

```diff
@@ -1,5 +1,22 @@
 FROM registry.redhat.io/rhscl/postgresql-12-rhel7
 
+# Install pg_repack
+USER root
+
+RUN yum -y install https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+RUN yum install rh-postgresql12-postgresql-devel \
```

Comment:
rh-postgresql12-* packages can't be installed after rhel 8 image switch

Reply:
should be good now, i also fixed the 400 return values